cmd/executor/cmd/root.go

```diff
@@ -84,8 +84,12 @@ var RootCmd = &cobra.Command{
 				return err
 			}
 
+			if err := checkNoDeprecatedFlags(); err != nil {
+				return errors.Wrap(err, "deprecated flags used")
+			}
```

Comment:
This still looks like it will fail the command if any deprecated flags are used... 🤔